dspace-api/src/test/java/org/dspace/builder/BundleBuilder.java

```diff
@@ -47,6 +48,11 @@ public BundleBuilder withName(String name) {
         return this;
     }
 
+    public BundleBuilder withPrimaryBitstream(Bitstream bitstream) {
+        this.primaryBitstream = bitstream;
+        return this;
+    }
+
```

Comment:
would be better to add an overloaded method `addBitstream` that takes a second parameter `boolean isPrimary` used to set the primary bitstream in the bundle?
in this way the api should look easier to use and there is no risk that someone would set a primary bitstream that is outside the bundle